src/tripCard/TripCard.tsx

```diff
@@ -172,7 +174,15 @@ const TripCard = ({
                 <div className="kirk-tripCard-avatar">
                   <Avatar image={driver.avatarUrl} />
                 </div>
-                <Text display={TextDisplayType.TITLE}>{driver.firstName}</Text>
+                <div>
+                  <Text display={TextDisplayType.TITLE}>{driver.firstName}</Text>
+                  {driver.rating && (
+                    <div className="kirk-tripCard-ratingContainer">
+                      <Star fill={1} size={16} />
```

Comment:
Maybe we can create a new case for the Rating component and use it here?

Reply:
I don't know, this case with only 1 star filled only exists in the component Profile / Search results (in sketch). I think we shouldn't make the Rating component more complex, since it's only used in the TripCard. WDYT?

Reply:
I have no strong opinion about this (hence why I used "maybe") so that's OK.

Another thing though is that after I looked at the design, I think there should be a little bit more space between the name and the rating

Reply:
Good catch, I'll double check the spacing

Reply:
Fixed & tested on SPA
![image](https://user-images.githubusercontent.com/1606624/71624930-54e4b000-2be5-11ea-9611-7b493ece802b.png)